nemo/collections/asr/models/rnnt_bpe_models.py

```diff
@@ -210,6 +210,14 @@ def list_available_models(cls) -> List[PretrainedModelInfo]:
             location="https://api.ngc.nvidia.com/v2/models/nvidia/nemo/stt_kab_conformer_transducer_large/versions/1.12.0/files/stt_kab_conformer_transducer_large.nemo",
         )
         results.append(model)
+
+        model = PretrainedModelInfo(
+            pretrained_model_name="stt_en_conformer_transducer_large",
```

Comment:
Please fix the pretrained model name: `_en_`=>`_be_`

Reply:
Done